public/index.html

```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.2/semantic.min.css"></link>
```

Comment:
I suggest you look into this.

https://semantic-ui.com/introduction/getting-started.html#install-semantic-ui

Assets will load faster since it will reside inside your server (of course, minifying it will improve the performance further, which is optional).